packages/graalvm/build.gradle.kts

```diff
@@ -127,6 +127,9 @@ dependencies {
   implementation(libs.micronaut.cache.core)
   implementation(libs.micronaut.cache.caffeine)
 
+  // Netty
+  implementation(libs.reactor.netty)
```

Comment:
need to swap for a more direct netty dependency (we're not really using reactor anymore, only netty)

Reply:
we have a netty bom which can be used with:
```
implementation(platform(libs.netty.bom))
```

then you can simply select whatever netty dep you want to use without declaring a version.